lib/presentation/shared/form/date_time_form_field.dart

```diff
@@ -103,20 +103,18 @@ class _CollactionDateTimeFormFieldState
 
   void _validateDateTime() {
     widget.validationCallback == null
-        ? _validationOutput = ValidationOutput(
-            error: false,
-            output: _dateTime,
-          )
+        ? _dateSet && _timeSet
+            ? _validationOutput = ValidationOutput(
+                error: false,
+                output: _dateTime,
+              )
+            : _validationOutput = ValidationOutput(error: true, output: null)
         : _validationOutput = widget.validationCallback!(
             _dateTime,
             _dateSet,
             _timeSet,
           );
```

Comment:
Some clarification regarding this validation logic. If there's no `validationCallback` passed to the field, means that it doesn't matter the validation, so fields could be empty or null. But with that change, if there's not a `validationCallback` there's still a validation, that if `_dateSet` && `_timeSet` are false an error is produced.

Reply:
Good catch, it should be `error: false` in line 111.